Serial protocol uses obsolete "UndoDB" commands

The Delve fork uses GDB serial protocol packets starting `vUndoDB`. But these packets no longer work: udbserver was changed to use `vUDB` in commit [30427c0608](https://git.undoers.io/Undo/core/commit/30427c060879f1b7735cba5d221d4a4667f99453) (October 2020). So UDB 6.2 and later are not supported. This manifests as test failures in `TestCheckpoints` and `TestRestart_breakpointPreservation` with UDB 6.2:
```
$ go run scripts/make.go test -v -s terminal -r TestCheckpoints -b undo
=== RUN   TestCheckpoints
live-record: Maximum event log size is 1G
Hello, World!
live-record: Saving to '/tmp/undo036918059'...
live-record: Termination recording written to /tmp/undo036918059
live-record: Detaching...
6.2.2 (build 72) udbserver
using inferred keyfile: /home/grees/core/oldrelease-6.2.2.72/key
Listening on port 44473
Connection established
--- FAIL: TestCheckpoints (1.75s)
    support.go:246: enabling recording for github.com/undoio/delve/pkg/terminal.TestCheckpoints
    command_test.go:581: "continue": "> main.main() /home/grees/github.com/delve/_fixtures/continuetestprog.go:16 (hits goroutine(1):1 total:1) (PC: 0x48d2df)\n    11:\t\n    12:\tfunc sayhi() {\n    13:\t\tfmt.Println(\"Hello, World!\")\n    14:\t}\n    15:\t\n=>  16:\tfunc main() {\n    17:\t\tsleepytime()\n    18:\t\tsayhi()\n    19:\t}\n"
    command_test.go:105: output of "checkpoint": ""
    command_test.go:106: Error executing <checkpoint>: unsupported packet $vUndoDB:get_time during undoCmd
$ go run scripts/make.go test -v -s integration -r TestRestart_breakpointPreservation -b undo
=== RUN   TestRestart_breakpointPreservation
live-record: Maximum event log size is 1G
Hello, World!
live-record: Saving to '/tmp/undo377250632'...
live-record: Termination recording written to /tmp/undo377250632
live-record: Detaching...
6.2.2 (build 72) udbserver
using inferred keyfile: /home/grees/core/oldrelease-6.2.2.72/key
Listening on port 44021
Connection established
undodb: Failed to communicate with front-end debugger. comms->down=4: -104: ECONNRESET (Connection reset by peer)
--- FAIL: TestRestart_breakpointPreservation (1.66s)
    support.go:246: enabling recording for github.com/undoio/delve/service/test_test.TestRestart_breakpointPreservation
    integration2_test.go:145: Restart
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x92c519]

goroutine 20 [running]:
testing.tRunner.func1(0xc00013c300)
	/home/grees/goroot/src/testing/testing.go:874 +0x3a3
panic(0x9d7ce0, 0xf350b0)
	/home/grees/goroot/src/runtime/panic.go:679 +0x1b2
github.com/undoio/delve/service/test_test.TestRestart_breakpointPreservation.func1(0xb7c1e0, 0xc00076c610)
	/home/grees/github.com/delve/service/test/integration2_test.go:149 +0x269
github.com/undoio/delve/service/test_test.withTestClient2.func1(0xb7c1e0, 0xc00076c610, 0xa9047c, 0x10, 0xc00013a120, 0x1e, 0xc00002c280, 0x3a, 0xc00002a420, 0x26)
	/home/grees/github.com/delve/service/test/integration2_test.go:49 +0x39
github.com/undoio/delve/service/test_test.withTestClient2Extended(0xa9047c, 0x10, 0xc00013c300, 0xc0003a3f10)
	/home/grees/github.com/delve/service/test/integration2_test.go:83 +0x13d
github.com/undoio/delve/service/test_test.withTestClient2(0xa9047c, 0x10, 0xc00013c300, 0xc000064f50)
	/home/grees/github.com/delve/service/test/integration2_test.go:48 +0x67
github.com/undoio/delve/service/test_test.TestRestart_breakpointPreservation(0xc00013c300)
	/home/grees/github.com/delve/service/test/integration2_test.go:131 +0x7d
testing.tRunner(0xc00013c300, 0xab6bd8)
	/home/grees/goroot/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/home/grees/goroot/src/testing/testing.go:960 +0x350
```

--- a/pkg/proc/gdbserial/gdbserver_conn.go
+++ b/pkg/proc/gdbserial/gdbserver_conn.go
@@ -990,7 +990,7 @@ func (conn *gdbConn) restart(pos string) error {
 
 	if conn.isUndoServer {
 		if pos != "" {
-			fmt.Fprint(&conn.outbuf, "$vUndoDB;goto_time;"+pos)
+			fmt.Fprint(&conn.outbuf, "$vUDB;goto_time;"+pos)
 		} else {
 			// Find the actual min BB count.
 			// TODO: is defaulting to zero if we can't get it correct?
@@ -1006,7 +1006,7 @@ func (conn *gdbConn) restart(pos string) error {
 			}
 
 			conn.outbuf.Reset()
-			fmt.Fprint(&conn.outbuf, "$vUndoDB;goto_bbcount;"+minBbCount)
+			fmt.Fprint(&conn.outbuf, "$vUDB;goto_bbcount;"+minBbCount)
 		}
 	} else {
 		fmt.Fprint(&conn.outbuf, "$vRun;")
@@ -1042,13 +1042,13 @@ func (conn *gdbConn) qRRCmd(args ...string) (string, error) {
 	return string(data), nil
 }
 
-// undoCmd executes a vUndoDB command
+// undoCmd executes a vUDB command
 func (conn *gdbConn) undoCmd(args ...string) (string, error) {
 	if len(args) == 0 {
 		panic("must specify at least one argument for undoCmd")
 	}
 	conn.outbuf.Reset()
-	fmt.Fprint(&conn.outbuf, "$vUndoDB")
+	fmt.Fprint(&conn.outbuf, "$vUDB")
 	for _, arg := range args {
 		fmt.Fprint(&conn.outbuf, ":", arg)
 	}

